database_cleanup/tests/test_database_cleanup.py

```diff
@@ -1,6 +1,7 @@
-# Copyright 2016 Therp BV <http://therp.nl>
```

Comment:
Should this be in other PR?

Reply:
there are a bunch of unrelated things that make this branch (and all other 13.0 PRs) red, so @NL66278 just fixes those here. As long as this happens in different commits, that's totally fine. When this will be merged, the whole branch will be green, and we can rebase the other PRs to also have CI succeed there and merge

Reply:
I need this to get travis green.